app/packages/embeddings/src/useExtendedStageEffect.tsx

```diff
@@ -23,11 +24,17 @@ export default function useExtendedStageEffect() {
 
   useEffect(() => {
     if (loadedPlot && Array.isArray(selection)) {
+      const shouldIncludeSelection = shouldResolveSelection(
+        view,
+        null,
```

Comment:
why is filter is `null` here? Can users lasso select on filters?

Reply:
The extended stage is either a

`fos.Mongo` - for using the lasso points to create a spatial query, which **does not** require `selection` (the sample or patch IDs)

or

`fos.Select`,  `fos.MatchLabels`, `fos.SelectBy` - which do require the selection to be present.

Note that we cannot use the presence of just the `pointsField` - since there are other requirements for allowing spatial querying such as when `SelectBy` is used to match the ids of a patch plot with each `patch.sample_id`.